src/FitterAlgoBase.cc

```diff
@@ -213,7 +213,7 @@ RooFitResult *FitterAlgoBase::doFit(RooAbsPdf &pdf, RooAbsData &data, const RooA
     nll0Value_ =  nll0;
     nllValue_ =  nll->getVal() - nll0;
     if (!ok && !keepFailures_) { std::cout << "Initial minimization failed. Aborting." << std::endl; return 0; }
-    if (doHesse) minim.minimizer().hesse();
+    if (doHesse) minim.hesse();
```

Comment:
For the analytic minimisation to work properly need to call the CascadeMinimizer hesse instead of directly via the RooMinimizer. In principle this could change existing results since the CascadeMinimizer also does the set/clear zero point calls on the NLL.